### DISCUSSION TITLE
Plugin tab is fully accessible from lock screen on the Steam Deck

### DISCUSSION THREAD

**Discussion by safijari** [CrankshaftForSteam/crankshaft#17] [Bug Report]
The plugin tab and all plugins contained therein can be accessed from the lock screen on the Steam Deck. This has the potential to be a security hole.

**Comment by coolavery** [comment]
Thanks for the report! Created a ticket on Sourcehut: https://todo.sr.ht/~avery/crankshaft/69

We'll be closing the Github issues on this repo and consolidating into Sourcehut tickets, I'll convert this to a discussion thread

**Comment by Olyve** [comment]
@safijari can you provide some steps on how you got into this scenario? I'd like to try and replicate the scenario

**Reply by DrakiaXYZ** [reply]
The only way I'm able to duplicate this is by putting the Deck to sleep while the plugin menu is open. When waking, it shows the plugin menu over top of the lock screen.

I'm assuming we can just close the plugin menu on sleep to resolve this

**Reply by Olyve** [reply]
Gotcha, that is what I was trying to figure out. And that makes sense to me, do we have a ticket to track this yet?

**Reply by safijari** [reply]
Another way to get there is to select "forgot pin" from the lock screen, go through the process of being logged out and getting back to the username/password entry screen. At that point the option to open the Steam Menu is present and the menu contains the plugins button. Full access to plugins is possible there.

**Reply by Olyve** [reply]
Good to know @safijari ! thanks for the info
